### ISSUE TITLE
Add documentation of unit test.

### ISSUE THREAD

**Issue by poojashresthacode** [ubsuny/23-Homework3G4#70]
@sharmistharanit 
I have edited your documentation file and I think the explanation of the unit test is missing there.

There are three unit tests. In one unit test, I have assumed that the final velocity of the object is zero and that's why the work done is negative in that case. Please have a look at it and give it a proper explanation saying that if the body comes to rest, the velocity will be zero, which means the body decelerates and the work done becomes negative.

Similarly, I have done the next unit test where the velocity is decreasing. Please explain the three-unit test by reviewing its input and examining its output.

**Comment by tirthbha** [comment]
@poojashresthacode What about the third unit test?

**Comment by poojashresthacode** [comment]
yeah, in the third one, I have used the decreasing velocities as input.
In the first unit test, I have used final velocity zero.
In  the second unit test, I have used initial and final velocity zero
